updater/lib/dependabot/dependency_change.rb

```diff
@@ -81,7 +81,7 @@ def should_replace_existing_pr?
       # NOTE: Gradle, Maven and Nuget dependency names can be case-insensitive
       # and the dependency name injected from a security advisory often doesn't
       # match what users have specified in their manifest.
-      updated_dependencies.map { |x| x.name.downcase } != job.dependencies.map(&:downcase)
+      updated_dependencies.map { |x| x.name.downcase }.uniq.sort != job.dependencies.map(&:downcase).uniq.sort
```

Comment:
Could you add a spec for this change? I have a feeling we'll be revisiting it again would be good to lock it down with a test.

Reply:
I added specs to cover this code.  Let me know if there's a better format to style them.